birdhouse/docker-compose.yml

```diff
@@ -92,6 +92,9 @@ services:
       - ${SSL_CERTIFICATE}:/opt/birdhouse/etc/nginx/cert.pem
     links:
       - mongodb
+    networks:
+      - default
+      - mongodb
```

Comment:
Phoenix also uses mongodb?  I though nobody was using mongodb before.  Note I do not know much about Phoenix.

Reply:
I'm not sure how it uses it, but it originally has a link to it. If it actually employs it behind the scene, it requires the network now.

I'm not sure if Phoenix is even relevant anymore in this stack.
It works with auth tokens that Twitcher we employ does not completely support nor validate with Magpie permissions. Most probably WPS operations done with it will fail unless the process is 100% public.

Reply:
I have a vague memory PCIC might use Phoenix.  Anyways, better be safe than sorry.  That network addition might not be needed, but it probably does no harm so good thing you add it.

Reply:
@fmigneault I just double check again before going live on prod with this PR (without Weaver activated) and I notice this.

The `mongodb` network is added to `phoenix` container but not to the `mongodb` container !  This looks really odd.  Is that accidental or intentional?

Reply:
Accidental. Must have happened during the many refactors into separate PRs.
I'll open another PR to add it.